### DISCUSSION TITLE
Vale a pena trocar de faculdade EAD para presencial?

### DISCUSSION THREAD

**Discussion by Gabriel-Tiburcio-Assis** [frontendbr/forum#1894] [Discussão]
Pessoal ,tenho 18 anos e  eu comecei uma faculdade EAD de Análise e Desenvolvimento de Sistemas em Agosto,porém não sei se estou no caminho certo permanecendo na faculdade.Moro em uma cidade onde a Federal daqui não tem um curso relacionado a tecnologia(a que tem.,que no caso é Recife, fica a mais de 2 horas da minha cidade e não seria viável me mudar(e no caso eu teria que prestar Enem daqui a um ano,pois não me escrevi para o desse ano) e na que estou cursando agora não tem me deixado com muito ânimo.As aulas que a faculdade possui são todas por meio de PDFs e possuem um vídeo de no máximo 4 minutos resumindo a matéria no final,o que eu já achei uma grande desvantagem,pois não sabia disso e não consigo estudar muito bem somente com PDFs.Por causa que eu entrei em Agosto,a faculdade colocou todos os alunos para cursar as matérias do 2° período,outro ponto negativo(comecei a ver POO sem saber lógica de programação(que por acaso era uma matéria do primeiro período,se não fosse pelas aulas do Guanabara eu não sei como estaria lidando com isso).Então fiquei em dúvida se devo continuar estudando nesse formato(o curso é de 2.5 anos) ,estudando com cursos por fora, ou se devo procurar uma faculdade privada presencial(essa instituição tem um polo presencial aqui na minha cidade,e por relatos de amigos meus,tem um ensino decente.Alguém poderia me ajudar,pois realmente não sei o que fazer.

**Comment by barraponto** [comment]
Networking é fundamental. Os cursos presenciais são maravilhosos nesse sentido, tanto no sentido profissional quanto no social mesmo. Se você não pode ir pra uma presencial, frequente eventos. Faça parte de comunidades profissionais, tanto remotamente quanto presencialmente. Assim vc tb permanece engajado nos tópicos do seu interesse e acaba aprendendo mais com gente que também está engajada.

Quanto ao diploma, não faz tanta diferença.

**Comment by yurimutti** [comment]
Então Gabriel também fiquei bem decepcionado! Comecei Análise e Desenvolvimento de Sistemas EAD em uma boa faculdade aqui da cidade e achei bom Ead vão ter aulas gravadas,apostilas etc mais para minha surpresa somente, pequenos links para youtube de terceiros e alguns PDFs. Fiquei pensando em desistir e continuar fazendo os cursos de desenvolvimento front end que já faco e cheguei a conclusão que o diploma não e tão relevante mais mesmo assim algumas vagas pedem e ate mesmo para facilidade de conseguir um estágio então como são 2.5 anos eu decidi continuar ate me formar tentando aprender o máximo, mais meu foco principal de conhecimento são os outros cursos que estou fazendo.
